client/app/scripts/superdesk-packaging/views/sd-add-package-dropdown.html

```diff
@@ -0,0 +1,7 @@
+<ul>
+    <li ng-repeat="group in groupList track by $index">
```

Comment:
Well, in such case you can simply track by the item itself, provided that items are unique. ;)

But never mind, let's not spend too much time on this anymore, no need to update just because of it. Only some `fit` needs to be removed to make the build pass, and we're done.